docs/guides/component-testing/third-party-definitions.mdx

```diff
@@ -0,0 +1,302 @@
+  `,
+})
+```
+
+Our Framework Definition shows up in Cypress! It has the "community" label,
+indicating it's a third party definition.
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-1.png"
+  alt="custom framework definition"
+/>
+
+We defined the `dependencies`, which are also correctly handled - we haven't
+installed them all, so Cypress is prompting us to do so:
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-2.png"
+  alt="required dependencies"
+/>
+
+## Mount Adapter
+
+The second part of defining a Framework Definition is the Mount Adapter. This is
+the function that renders the component in your tests using `cy.mount()`.
+
+By default, Cypress will look for this as a `mount` function that is a **named
+export** from the package. This should be written in a `index.mjs` file. This
+example is for a [Solid.js](https://www.solidjs.com/) mount adapter:
+
+```js
+import { getContainerEl, setupHooks } from '@cypress/mount-utils'
+import { render } from 'solid-js/web'
+
+let dispose
+
+function cleanup() {
+  dispose?.()
+}
+
+/**
+ * @param {() => JSX.Element} - component to render
+ */
+export function mount(component, options = {}) {
+  // Retrieve root DOM element that Cypress has prepared for this test
+  const root = getContainerEl()
+
+  dispose = render(() => component, root)
+
+  // Wait until next microtick to ensure any async render logic has executed
+  return cy.wait(0, { log: false }).then(() => {
+    if (options.log) {
```

Comment:
This won't work unless we check the value of `options.log`. We're assuming that it's true here but that's not the case unless it is explicitly set. Should we change this to

```suggestion
    if (options.log !== false) {
```